test/getDeployedEntitiesStream.spec.ts

```diff
@@ -125,13 +122,199 @@ test('getDeployedEntitiesStream', ({ components, stubComponents }) => {
       { entityType: 'profile', entityId: 'Qm000013', localTimestamp: 13, authChain, pointers: ['0x1'] },
     ])
   })
+})
+
+test('fetches a stream without deleting the downloaded file', ({ components, stubComponents }) => {
```

Comment:
why do you need to not delete the file?

Reply:
Don't know. Was an old test, i just moved it